jboss-eap-xp1-openjdk8-openshift.json

```diff
@@ -0,0 +1,123 @@
+{
+    "kind": "List",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "jboss-eap-xp1-openjdk8-openshift",
+        "annotations": {
+            "description": "ImageStream definition for JBoss Enterprise Application Platform expansion pack with OpenJDK 8",
+            "openshift.io/provider-display-name": "Red Hat"
+        }
+    },
+    "items": [
+        {
+            "kind": "ImageStream",
+            "apiVersion": "v1",
+            "metadata": {
+                "name": "jboss-eap-xp1-openjdk8-openshift",
+                "annotations": {
+                    "openshift.io/display-name": "JBoss EAP XP with OpenJDK 8",
+                    "openshift.io/provider-display-name": "Red Hat",
+                    "version": "1.0"
+                }
+            },
+            "labels": {
+                "xpaas": "1.0"
+            },
+            "spec": {
+                "tags": [
+                    {
+                        "name": "latest",
+                        "annotations": {
+                            "description": "The latest released build of JBoss EAP XP",
```

Comment:
should be "JBoss EAP expansion pack 1.0 image for OpenShift to build and run Microservices applications on RHEL7 with OpenJDK8  "